compiler/src/iree/compiler/Codegen/Utils/CPUUtils.cpp

```diff
@@ -63,4 +66,22 @@ FailureOr<Operation *> getRootOperation(ArrayRef<Operation *> computeOps) {
   return rootOperation;
 }
 
+StringAttr getEnableLoopPeelingAttrName(MLIRContext *ctx) {
+  return StringAttr::get(ctx, kLoopPeelingAttrName);
+}
+
+bool isLoopPeelingEnabled(FunctionOpInterface funcOp) {
+  auto context = funcOp->getContext();
+
+  DictionaryAttr config = getTranslationInfo(funcOp).getConfiguration();
+
+  if (!config) {
+    return false;
+  }
+
+  auto enableLoopPeelingAttrName = getEnableLoopPeelingAttrName(context);
+  auto enableLoopPeelingAttr = config.getNamed(enableLoopPeelingAttrName);
+  return enableLoopPeelingAttr.has_value() ? true : false;
```

Comment:
nit: There's no need to check a unit attribute (simply having the attribute enables peeling). Also, you don't need to construct a `StringAttr` to check (`.contains` can take a `StringRef`).
```suggestion
  return config.contains(kLoopPeelingAttrName);
```

Reply:
nit I just noticed: `kLoopPeelingAttrName` -> `kEnableLoopPeelingAttrName`